plasma_framework/test/endToEndTests/Extendiblility.e2e.test.js

```diff
@@ -1,312 +0,0 @@
-const EthVault = artifacts.require('EthVault');
```

Comment:
Do we really want to delete this entire test? Now we have no e2e test for extendibility 😞

Reply:
I am thinking to compensate that with the fee transaction e2e coming soon. So it would be payment v1 -> payment v2 + fee.

Reply:
hmmmm......so asked the fee team and seem like we are more on the side of not needing payment v2 for fee. sigh....any cool idea for extendibility test?

Reply:
@boolafish Let's just merge this and add a new extensibility test later. I don't want to hold up this PR any longer 😄

Reply:
Thanks :)